aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -65,36 +65,36 @@ crate::params::define_gas_parameters!(
         [
             gas_unit_scaling_factor: GasScalingFactor,
             "gas_unit_scaling_factor",
-            1000
+            10_000
```

Comment:
Why this change?

Reply:
After a discussion with Avery, I realized that it's important we preserve some headroom for future appreciation of the coin. With the scaling factor being set to 10000, a p2p transfer transaction costs ~700, which has enough precision.